crates/re_types_builder/src/codegen/rust/deserializer.rs

```diff
@@ -81,6 +81,7 @@ pub fn quote_arrow_deserializer(
             obj_field.is_nullable,
             obj_field_fqname,
             &data_src,
+            false,
```

Comment:
nit: This would be more readable with an enum, or as a struct member (e.g. `DeserOptions { is_list_inner: true }`). That would also give you a place to document it (`is_list_inner` is an argument, and not documented). I think this code is quite complex, and more docstrings wouldn't hurt.